ReadMe.md

```diff
@@ -104,7 +92,7 @@ ADIOS2 build configuration:
 
 -- Configuring done
 -- Generating done
--- Build files have been written to: /home/chuck/adios2/build
+-- Build files have been written to: /home/chuck/ADIOS2/build
```

Comment:
`/home/chuck/ADIOS2/build` -> `/home/chuck/adios2_build`

Reply:
Is the cmake output necessary? It seems reasonable to get rid of it all.

Reply:
Necessary, no, helpful, yes.  The documentation shows an abbreviated output to inform users the kind of information they can expect to see.  Namely things like the compiler id in the beginning and option summary and the end let the user know they have a sane configuration.  That being said, I'm fine too if instead of fixing the path in the output you want to just remove that last line "Build files have been written to..." from the output in the docs as it's not particularly informative like the rest of it is.  Up to you.